opcua/common/instantiate.py

```diff
@@ -86,7 +89,8 @@ def _instantiate_node(server, parentid, rdesc, nodeid, bname, dname=None, recurs
                         nodeids = _instantiate_node(server, res.AddedNodeId, c_rdesc, nodeid=ua.NodeId(identifier=inst_nodeid, namespaceidx=res.AddedNodeId.NamespaceIndex), bname=c_rdesc.BrowseName)
                     else:
                         nodeids = _instantiate_node(server, res.AddedNodeId, c_rdesc, nodeid=ua.NodeId(namespaceidx=res.AddedNodeId.NamespaceIndex), bname=c_rdesc.BrowseName)
-                    added_nodes.extend(nodeids)
+                    if nodeids is not None:
+                        added_nodes.extend(nodeids)
```

Comment:
Look in code of that method, it might be missing support for DataType...
then maybe we should raise an exception, not only show a warning....